### ISSUE TITLE
auto_compression/pytorch_yolo_series压缩得到的模型大小没有变化

### ISSUE THREAD

**Issue by Ledgero** [PaddlePaddle/PaddleSlim#1646]
版本：
cuda :11.3 cudnn:8.6.0 (对应cuda11.x) python :3.8 paddlepaddle-gpu: 2.4.1.post112 paddleslim:2.4.1
系统：linux

**问题：**
运行run.py压缩yolov7-tiny , 原模型大小是24.9，压缩后导出的onnx模型大小还是24.9.同时精度没有发生改变。

运行参数：
`Global:
  model_dir: ./yolov7-tiny.onnx
  image_path: None      # If image_path is set, it will be trained directly based on unlabeled images, no need to set the COCO dataset path.
  coco_dataset_dir: dataset/coco/
  coco_train_image_dir: train2017
  coco_train_anno_path: annotations/instances_train2017.json
  coco_val_image_dir: val2017
  coco_val_anno_path: annotations/instances_val2017.json
  arch: YOLOv7

Distillation:
  alpha: 1.0
  loss: soft_label

Quantization:
  onnx_format: True
  activation_quantize_type: 'moving_average_abs_max'
  quantize_op_types:
  - conv2d
  - depthwise_conv2d

TrainConfig:
  train_iter: 5000
  eval_iter: 1000
  learning_rate:
    type: CosineAnnealingDecay 
    learning_rate: 0.00003
    T_max: 8000
  optimizer_builder:
    optimizer:
      type: SGD
    weight_decay: 0.00004`
运行中仅报以下warming：
**WARMING:root:canot import name 'layers' from 'parl'(/home/zjJl/anaconda3/envs/paddLe/Llib/python3.8/site-packages/parl/..init...py)
/hone/zj]l ancondas/envs/pettle/Lib/python5.8/site-packagesl_tistutils_hackl ..init ..py.83: Userlanning: Setuptools is replacing distutils.warnings.warn("Setuptools is replacing distutils.")**

**Comment by zzjjay** [comment]
该问题是没有开启量化训练导致的。
在paddleslim2.4.1中 需要将量化配置的关键词【Quantization】改为【QuantAware】
可参考最新[自动压缩超参文档](https://github.com/PaddlePaddle/PaddleSlim/blob/develop/example/auto_compression/hyperparameter_tutorial.md#111-%E9%87%8F%E5%8C%96%E8%AE%AD%E7%BB%83quant-aware)进行配置。

**Comment by Ledgero** [comment]
> 该问题是没有开启量化训练导致的。 在paddleslim2.4.1中 需要将量化配置的关键词【Quantization】改为【QuantAware】 可参考最新[自动压缩超参文档](https://github.com/PaddlePaddle/PaddleSlim/blob/develop/example/auto_compression/hyperparameter_tutorial.md#111-%E9%87%8F%E5%8C%96%E8%AE%AD%E7%BB%83quant-aware)进行配置。

@zzjjay 感谢，目前模型的pdiparams已从24.9M压缩到6.4M,但是导出的onnx大小还是没有变化（依旧24.9M）,想请问一下这是什么原因？
